lib/fastlane/actions/add_icon_overlay.rb

```diff
@@ -0,0 +1,181 @@
+                                       env_name: "FL_ADD_ICON_OVERLAY_OVERLAY_IMAGE_PATH",
+                                       description: "Path for the image that will be overlaid on the icons",
+                                       is_string: true,
+                                       optional: true,
+                                       verify_block: Proc.new do |value|
+                                       raise "No overlay image path for AddIconOverlayAction given, pass using `overlay_image_path: 'image_path.png'`".red unless (value and not value.empty?)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :overlay_type,
+                                       env_name: "FL_ADD_ICON_OVERLAY_OVERLAY_TYPE",
+                                       description: "Flag specifying if an image, text or banner will be overlayed over the icon",
+                                       is_string: true,
+                                       default_value: 'text',
+                                       verify_block: Proc.new do |value|
+                                         raise "Invalid overlay type provided. Expected 'text' or 'image' and '#{value}' was provided" unless (value == "text" or value == "image" or value == "banner")
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :text_background_color,
+                                       env_name: "FL_ADD_ICON_OVERLAY_TEXT_BACKGROUND_COLOR",
+                                       description: "Background color for the text overlay. If a hex value is provided must contain the '#' character",
+                                       is_string: true,
+                                       default_value: "#C2C2C3",
+                                       optional: true
+                                      ),
+          FastlaneCore::ConfigItem.new(key: :text_foreground_color,
+                                       env_name: "FL_ADD_ICON_OVERLAY_TEXT_FOREGROUND_COLOR",
+                                       description: "Foreground color for the text overlay (Text color). If a hex value is provided must contain the '#' character",
+                                       is_string: true,
+                                       default_value: "white",
+                                       optional: true
+                                      ),
+          FastlaneCore::ConfigItem.new(key: :text_overlay_height,
+                                       env_name: "FL_ADD_ICON_OVERLAY_OVERLAY_HEIGHT",
+                                       description: "Height of the overlay in respect to the overall height of the icon (e.g. 0.5 for 50%)",
+                                       optional: true,
+                                       default_value: 0.3,
+                                       is_string: false,
+                                       verify_block: Proc.new do |value|
```

Comment:
Already fixed with e0e2bb3f35cfd00ad9401eea4a7e4b72dffcb87a but `HoundCI` failed to pick the change